### DISCUSSION TITLE
Post installation steps for Obsrv sandbox

### DISCUSSION THREAD

**Discussion by allwin-winfred** [Sunbird-Obsrv/Community#157] [Installation] [answered]
Hello team, 

We have completed the installation and have ran the postman APIs. All the APIs are working fine now. 
What are the next steps to configuration to  showcase the sandbox for end users.

**Comment by sowmya-dixit** [comment, marked as answer]
@allwin-winfred We are glad to hear that the installation is completed successfully. Now you can,

- You can create new datasets specific to end user usecases referring to example datasets. 
- Ingest some data for the created datasets through dataIN API. - https://obsrv.sunbird.org/guides/data-in-and-out-apis
- Query for the ingested data using query APIs.
- Also create some charts using superset for better visualizations for end users.

**Comment by allwin-winfred** [comment]
@sowmya-dixit Noted Thanks.
